Godeps/Godeps.json

```diff
@@ -1,7 +1,7 @@
 {
 	"ImportPath": "github.com/snowplow/sql-runner",
-	"GoVersion": "go1.7",
-	"GodepVersion": "v76",
+	"GoVersion": "go1.9",
```

Comment:
Really - did bumping to 1.9 not change any of the deps that are vendored in?

Reply:
No, I didn't. You probably missed something.  There are some new deps in vendor.

Reply:
Not in this commit there aren't: https://github.com/snowplow/sql-runner/pull/115/commits/458abf70462e48a9ec67d32293b18625d946d6c1